[v3.0.0] schemaLegacy.d.ts missing in package

**React version** (e.g., 16.11.0)

**Concurrent mode** no

**Describe the bug**

It appears that `/src/resource/schemaLegacy.d.ts` is not copied to the output folder ([browse the package files on yarnpkg.com](https://yarnpkg.com/en/package/rest-hooks?files)), making TypeScript build fail since it's referenced in `/src/resource/types.ts`.

```
$ tsc
../../../../node_modules/rest-hooks/lib/resource/types.d.ts:2:84 - error TS2307: Cannot find module './schemaLegacy'.

2 import { SchemaList as SchemaListLegacy, SchemaDetail as SchemaDetailLegacy } from './schemaLegacy';
                                                                                     ~~~~~~~~~~~~~~~~
```

**To Reproduce**

**Expected behavior**
All necessary files should be present in published package, or the build will fail.

**Additional context**
The TypeScript compiler doesn't copy *d.ts files to the output filder by default: https://stackoverflow.com/questions/56018167/typescript-does-not-copy-d-ts-files-to-build

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "build:types": "ttsc --emitDeclarationOnly -p ./src",
     "build:bundle": "rollup -c",
     "build:clean": "rimraf lib",
-    "build": "npm run build:lib && npm run build:types && cpy ./src/resource/normal.d.ts ./lib/resource",
+    "build": "npm run build:lib && npm run build:types && cpy ./src/resource/schemaLegacy.d.ts ./lib/resource",
     "dev": "yarn run build:lib -w",
     "prepare": "npm run build:clean && npm run build",
     "prepublishOnly": "npm run build:bundle",

